qase-python-commons/src/qaseio/commons/testops.py

```diff
@@ -335,24 +345,32 @@ def _send_result(self, result: Result):
             if result.get_suite_title():
                 case_data["suite_title"] = "\t".join(result.get_suite_title().split("."))
 
+            result_model = ResultCreate(
```

Comment:
At line 141 `result_model` is a `Map` object, and here it's `ResultCreate`.

Reply:
We have fixed it in the new version.